types/three/examples/jsm/loaders/KTX2Loader.d.ts

```diff
@@ -1,17 +1,18 @@
-import { LoadingManager, CompressedTextureLoader, CompressedTexture, WebGLRenderer } from '../../../src/Three.js';
+import { LoadingManager, Loader, CompressedTexture, WebGLRenderer } from '../../../src/Three.js';
 import WebGPURenderer from '../renderers/webgpu/WebGPURenderer.js';
 
-export class KTX2Loader extends CompressedTextureLoader {
+export class KTX2Loader extends Loader {
```

Comment:
The `Loader` generic needs to be set for this to work:

```suggestion
export class KTX2Loader extends Loader<CompressedTexture> {
```